staging/src/k8s.io/apiserver/pkg/server/options/authentication.go

```diff
@@ -170,6 +173,19 @@ func (s *DelegatingAuthenticationOptions) ApplyTo(c *server.Config) error {
 		return fmt.Errorf("unable to load client CA file: %v", err)
 	}
 
+	if s.SkipInClusterLookup {
```

Comment:
I'm not seeing why this is needed... ToAuthenticationConfig -> DelegatingAuthenticatorConfig#New already sets up an x509 authenticator, right?

Reply:
The delegating authenticator needs a couple of more config flags. At least, I got it running without the authentication-skip-lookup flags:

```
sample-apiserver --secure-port 6443 --etcd-servers http://127.0.0.1:2379 --client-ca-file ca.crt --authorization-skip-lookup --kubeconfig ~/.kube/config --authentication-kubeconfig ~/.kube/config --v=7 --requestheader-client-ca-file ca.crt
```
I.e by reusing the apiserver ca.crt and by pointing the authn kubeconfig to `~/.kube/config`. I guess if we document this in the README.md we can live without the new flag.